benefits/static/css/admin/styles.css

```diff
@@ -0,0 +1,124 @@
+  --form-field-text-line-height: 1.25; /*Displayed as 125% on Figma */
+  --h4-line-height: 1.2; /* Displayed as 120% on Figma */
+  --letter-spacing-2: 0.02; /* Displayed as 2% on Figma */
+  --letter-spacing-3: 0.03; /* Displayed as 3% on Figma */
+  --letter-spacing-5: 0.05; /* Displayed as 5% on Figma */
+  --dark-color: #212121;
+  --hover-color: #044869;
+  --error-color: #ea1010;
+  --selected-color: #562b97;
+  --focus-color: #fdb714;
+  --secondary-bg-color: #fafafa;
+  --error-color-rgb: 234, 16, 16;
+  --bs-danger: var(--error-color);
+  --bs-danger-rgb: var(--error-color-rgb);
+  --standout-color: #323a45;
+  --h1-text-align: left;
+  --h3-font-size: calc(20rem / 16);
+  --font-size-24px: calc(24rem / 16);
+  --font-size-20px: calc(20rem / 16);
+  --font-size-18px: calc(18rem / 16);
+  --font-size-15px: calc(15rem / 16);
+  --font-size-16px: calc(16rem / 16);
+  --font-size-14px: calc(14rem / 16);
+  --font-size-12px: calc(12rem / 16);
+  --border-width: calc(2rem / 16);
+  --border-radius: calc(3rem / 16);
+  --focus-style: 3px solid var(--focus-color);
+  --focus-border-radius: calc(2rem / 16);
+  --focus-outline-offset: calc(2rem / 16);
+  --focus-border-radius: calc(2rem / 16);
+}
+
+@font-face {
+  font-family: "Public Sans";
+  font-weight: var(--bold-font-weight);
+  font-style: normal;
+  /* prettier-ignore */
+  src: local("PublicSans"), url("../../fonts/PublicSans-Bold.woff") format("woff");
+  /* prettier-ignore */
+}
+
+/* Buttons */
+/* Primary Button: Use all three classes: btn btn-lg btn-primary */
+/* Set button width in parent with Bootstrap column */
+/* Height: 60px on Desktop; 72 on mobile*/
+
+:root {
+  --primary-button-padding: 19px 0;
+}
+
+@media (min-width: 992px) {
+  :root {
+    --primary-button-padding: 13px 0;
+  }
+}
+
+.btn.btn-lg.btn-primary {
+  background-color: var(--primary-color);
+  border-color: var(--primary-color);
+  border-width: 2px;
+  font-weight: var(--medium-font-weight);
+  font-size: var(--font-size-20px);
+  letter-spacing: calc(var(--font-size-20px) * var(--letter-spacing-2));
+  width: 100%;
+  padding: var(--primary-button-padding);
+}
+
+.btn.btn-lg.btn-primary:hover {
+  background-color: var(--hover-color);
+  border-color: var(--hover-color);
+}
+
+.btn.btn-lg.btn-primary:focus,
+.btn.btn-lg.btn-primary:focus-visible,
+.btn.btn-lg:focus,
+.btn.btn-lg:focus-visible,
+.btn-outline-dark:focus,
+.btn-outline-dark:focus-visible,
+.btn-outline-light:focus,
+.btn-ouline-dark:focus-visible {
+  outline: 3px solid var(--focus-color) !important;
+  outline-offset: 2px !important;
+  box-shadow: none !important; /* override CA State Web Template */
+  text-decoration: none;
+}
+
+.btn-text {
+  font-weight: var(--medium-font-weight);
+  font-size: var(--font-size-20px);
+  letter-spacing: calc(var(--font-size-20px) * var(--letter-spacing-2));
+}
+
+:root {
+  --branding-title-color: #ffffff;
+}
+.branding-title {
+  color: var(--branding-title-color);
+}
+
```

Comment:
You can also just use `text-white` utility class from Bootstrap so we don't have to have this one-off style. Documented here: https://getbootstrap.com/docs/5.3/utilities/colors/#colors